crt_portal/static/sass/custom/landing.scss

```diff
@@ -174,6 +186,7 @@ $arrow-height: 24px;
     a,
     a:visited {
       color: white;
+      margin-top: 0;
```

Comment:
`a` tags should be inline elements and as such shouldn't be affected by top or bottom margin. Was there a particular `a` tag you were trying to target here?

Reply:
So regarding the margin-top, I added that in to handle the 24px space above the "Start a Report" button. 
![image](https://user-images.githubusercontent.com/66343959/88087062-590ef580-cb56-11ea-825c-75d9abff9477.png)
Without the margin-top:0 the button has an 8px margin applied. For my re-requested review and last commit I reverted this change. However I did want to explain my logic and would welcome your feedback.